index.html

```diff
@@ -0,0 +1,499 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Smashing magazine</title>
+  <link rel="stylesheet" href="assets/css/style.css">
+</head>
+
+<body class="total-width">
+  <header class="d-flex header fun-border">
+    <nav class="d-flex">
+    
+        <a class="logo-div bg-dark" href=""></a>
+      
+      <ul class="header-list d-flex">
+        <a class="list-item bg-medium" href="#"></a>
+        <a class="list-item bg-medium" href="#"></a>
+        <a class="list-item bg-medium" href="#"></a>
+        <a class="list-item bg-medium" href="#"></a>
+        <a class="list-item bg-medium" href="#"></a>
+        <a class="list-item bg-medium" href="#"></a>
+        <a class="list-item bg-medium" href="#"></a>
+        <a class="list-item bg-medium" href="#"></a>
+          <a class="list-item bg-dark" href="#"></a>
+      </ul>
```

Comment:
Semantically, the content of `ul` elements should have `li` elements: Your code should look like this:
```
<ul class="header-list d-flex">
        <li><a class="list-item bg-medium" href="#"></a></li>
       .....
</ul>
```